src/Bridge/Doctrine/Orm/Util/IdentifierManagerTrait.php

```diff
@@ -39,6 +39,8 @@
      */
     private function normalizeIdentifiers($id, ObjectManager $manager, string $resourceClass): array
     {
+        @trigger_error(sprintf('Trait ApiPlatform\Core\Bridge\Doctrine\Orm\Util\IdentifierManagerTrait will be removed in version 3.0. Using it is deprecated since 2.2.'), E_USER_DEPRECATED);
```

Comment:
Nahh I can't as "php-cs-fixer" is using `self` instead but because it's a trait the resulting class for `self::class` is wrong :|. IMO it's a bug in `php-cs-fixer`.

Reply:
Can't you just disable this rule for now?